app/views/partners/profiles/step/_agency_information_form.html.erb

```diff
@@ -3,18 +3,20 @@
 </div>
 
 <%= f.fields_for :profile, profile do |pf| %>
-  <div class="form-group">
-    <%= pf.input :agency_type,
-    # Symbolize keys, because simple_form will only do a I18n lookup for
-    # symbols
-      collection: Partners::Profile.agency_types_for_selection,
-      label: "Agency Type",
-      class: "form-control",
-      wrapper: :input_group %>
-  </div>
-
-  <div class="form-group">
-    <%= pf.input :other_agency_type, label: "Other Agency Type", class: "form-control" %>
+  <div data-controller="hide-by-source-val"
+       data-hide-by-source-val-values-to-hide-value='["career","abuse","bnb","church","college","cdc","health","outreach","legal","crisis","disab","district","domv","ece","child","edu","family","food","foster","govt","headstart","homevisit","homeless","hosp","infpan","lib","mhealth","military","police","preg","presch","ref","es","hs","ms","senior","tribal","treat","twoycollege","wic"]'>
```

Comment:
This seems backwards... can we update the controller to also allow us to only specify the ones we want to show? We shouldn't have to specify every single value _except_ the one we care about.